packages/config/src/ProfileInfo.ts

```diff
@@ -560,6 +576,31 @@ export class ProfileInfo {
         return this.mUsingTeamConfig;
     }
 
+    /**
+     * Load value of secure argument from the vault.
+     * @param arg Secure argument object
+     */
+    public loadSecureArg(arg: IProfArgAttrs): any {
```

Comment:
During final cleanup of this PR, I ask that you add an example of using this public function to the pseudocode within the block comments for the ProfileInfo class.